assets/src/edit-story/utils/usePreventWindowUnload.js

```diff
@@ -50,8 +50,8 @@ function usePreventWindowUnload() {
     },
     [context]
   );
-
   return setPreventUnload;
 }
 
-export default usePreventWindowUnload;
+const shouldDisablePrevent = Boolean(process.env.DISABLE_PREVENT);
```

Comment:
It would be good to document this somewhere. WDYT? Maybe under the [docs](https://github.com/google/web-stories-wp/tree/master/docs) folder create an `ENV_VARS.md` file?

Reply:
Added some info in https://github.com/google/web-stories-wp/pull/1176/commits/3b4c00742ec2c3d6e3dac88a31404f69c1bc5266